From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Todd Kjos <tkjos@google.com>
Date: Wed, 27 Jan 2021 11:43:47 -0800
Subject: ANDROID: use ANDROID_OEM_DATA for OEM data

Change vendor fields to OEM fields by using
ANDROID_OEM_DATA when the vendor fields were
originally requested by an OEM.

Bug: 177481081
Signed-off-by: Todd Kjos <tkjos@google.com>
Change-Id: Iaf1e80e07bfd78efb5a9b7ff5894ff751f272f23
---
 include/linux/sched.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/sched.h b/include/linux/sched.h
--- a/include/linux/sched.h
+++ b/include/linux/sched.h
@@ -1389,6 +1389,7 @@ struct task_struct {
 	struct callback_head		mce_kill_me;
 #endif
 	ANDROID_VENDOR_DATA_ARRAY(1, 64);
+	ANDROID_OEM_DATA_ARRAY(1, 2);
 
 #ifdef CONFIG_KRETPROBES
 	struct llist_head               kretprobe_instances;
